elifecrossref/funding.py

```diff
@@ -1,11 +1,15 @@
 from xml.etree.ElementTree import SubElement
 
 
+def do_funding(poa_article):
```

Comment:
these `do_*` functions appear to be simple predicates. Perhaps `has_*` would be better.

For example, `if has_funding(poa_article): ...`

Reply:
Thanks for the suggestion on the `do_` naming. I've run through a few alternatives and I'm not sure they're too much better, the definition is basically `should I do this? ___`. It could be something like `can_`, `approve_to_add_` or something like that. For now, I might stick with `do_`, since there are a few of these already and they follow a theme.

These are a way to encapsulate the minimum data input conditions of whether to add a particular XML part, and it's an easy way to protect against adding blank parent XML tags, when there's no data to populate its child tags.

Reply:
it's a function, it doesn't know or care about how it's going to be used. Looking at a `do_*` function that doesn't do anything besides return `True` or `False` is weird.

Reply:
> that doesn't do anything

by which I mean, it doesn't have side effects. It doesn't modify it's input in-place, it doesn't write to the screen, or write to a file or make a network call or change the application state.